ui/ducks/send/swap-and-send-utils.ts

```diff
@@ -132,9 +132,9 @@ const QUOTE_VALIDATORS = [
   },
   {
     property: 'error',
-    type: 'string',
-    validator: (error: string | null) =>
-      error === null || typeof error === 'string',
+    type: 'object',
+    validator: (error: any) =>
+      error === null || typeof error === 'object',
```

Comment:
The linter is also complaining about a new line here; there may be a broken setting in your VSCode